test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6252,7 +6253,33 @@ def submit(self, obj, script=None, extend=None, submit_dir=None):
         rc = None
 
         if isinstance(obj, Job):
-            if script is None and obj.script is not None:
+            if self.platform == 'cray' or self.platform == 'craysim':
+                m = None
+                vncompute = None
+                if 'Resource_List.select' in obj.attributes:
+                    m = re.match('.*cray_compute.*',
```

Comment:
I think its safe to search for '.\*vntype=cray_compute.\*' instead '.\*cray_compute.*'